### PR TITLE
added French Translations

### PR DESCRIPTION
**Pull request by yed30**
Hello ! 
I just translated the language files in French and organized them in a fr sub-directory in the translations one.

I hope this helps.
Regards

### CODE REVIEW

**Comment by yed30** [comment]
Well the build failed, but not because of my commit :)

**Comment by andrerom** [comment]
Cool, french review @bdunogier @yannickroger

File: bundle/Resources/translations/fr/ezrepoforms_content_type.fr.xlf [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+                <source>content_type.description</source>
+                <target>Description</target>
+            </trans-unit>
+            <trans-unit id="4">
+                <source>content_type.name_schema</source>
+                <target>Gabarit de nom de contenu</target>
+            </trans-unit>
+            <trans-unit id="5">
+                <source>content_type.url_alias_schema</source>
+                <target>Gabarit de nom d'alias d'URL</target>
```
Comment (bdunogier) [inline bundle/Resources/translations/fr/ezrepoforms_content_type.fr.xlf]: Gabarit d'alias d'URL ?

File: bundle/Resources/translations/fr/ezrepoforms_content_type.fr.xlf [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+            </trans-unit>
+            <trans-unit id="6">
+                <source>content_type.is_container</source>
+                <target>Container</target>
```
Comment (bdunogier) [inline bundle/Resources/translations/fr/ezrepoforms_content_type.fr.xlf]: Conteneur ?

**Comment by yed30** [comment]
Hello Bertrand,
I guess you are right, I was't sure myself of those ones and will make the changes you propose and push them tomorrow (Paris Time). Thanks a lot for your feeds back :)

**Comment by yed30** [comment]
I pushed the two changes :)

File: bundle/Resources/translations/fr/validators.fr.xlf [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                <source>ez.field_definition.default_field_value</source>
+                <target>La valeur par défaut du champ n'est pas valide.</target>
+            </trans-unit>
+            <trans-unit id="b4bcdedc4147dd912dc0247cea16c750" resname="ez.role.identifier.unique">
+                <source>ez.role.identifier.unique</source>
+                <target>L'identifiant "%identifier%" existe déjà. L'identifiant de rôle doit être unique.</target>
+            </trans-unit>
+            <trans-unit id="5387d26cceb54294dafe82be9b0d61d0">
+                <source>ez.section.identifier.format</source>
+                <target>Les identifiants de section ne peuvent contenir que les lettres "a" à "z", des chiffres et des underscores.</target>
```
Comment (bdunogier) [inline bundle/Resources/translations/fr/validators.fr.xlf]: "underscore" => "blanc souligné" ?

File: bundle/Resources/translations/fr/validators.fr.xlf [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            </trans-unit>
+            <trans-unit id="3782ffaac11648e8ead7248adb7a42bc">
+                <source>ez.section.identifier.unique</source>
+                <target>Une section avec cet identifiant existe déjà, merci d'en saisir un différent</target>
```
Comment (bdunogier) [inline bundle/Resources/translations/fr/validators.fr.xlf]: Point à la fin de la phrase ?

File: bundle/Resources/translations/fr/ezrepoforms_section.fr.xlf [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+                <source>section.id.label</source>
+                <target>ID de Section:</target>
+            </trans-unit>
+            <trans-unit id="60ff6d8b447be276d65797a5244d79c3">
+                <source>section.error.id_already_exist</source>
+                <target>Une section avec cet ID existe déjà, merci d'en saisir un différent.</target>
+            </trans-unit>
+            <trans-unit id="a07809a8014c979ffe3e2256610b0ea4">
+                <source>section.error.cannot_delete_with_assigned_content</source>
+                <target>Impossible de supprimer une section avec du contenu assigné.</target>
```
Comment (bdunogier) [inline bundle/Resources/translations/fr/ezrepoforms_section.fr.xlf]: "une section assignée à du contenu" ?

**Comment by bdunogier** [comment]
Hi there. A bit late, but we'll merge this. Thank you !

**Pull request merged**